docs/src/Fulma/Modifiers/Typography.fs

```diff
@@ -0,0 +1,83 @@
+module Modifiers.Typography
+
+open Fable.Helpers.React
+open Fable.Helpers.React.Props
+open Fulma
+
+let alignment () =
+    div [ Class "block" ]
+        [ Text.p [ Modifiers [ Modifier.TextAlignment (Screen.All, TextAlignment.Left) ] ]
+            [ str "Left" ]
+          hr [ ]
+          Text.p [ Modifiers [ Modifier.TextAlignment (Screen.All, TextAlignment.Centered) ] ]
+            [ str "Centered" ]
+          hr [ ]
+          Text.p [ Props [ Style [ Width "200px" ] ]
+                   Modifiers [ Modifier.TextAlignment (Screen.All, TextAlignment.Justified) ] ]
+            [ str "This text should be justified, this is why I am writing a long text. I am doing that in order to make it easier to see." ]
+          hr [ ]
+          Text.p [ Modifiers [ Modifier.TextAlignment (Screen.All, TextAlignment.Right) ] ]
+            [ str "Right" ] ]
+
+let transformation () =
+    Content.content [ ]
+        [ li [ ]
+            [ Text.span [ Modifiers [ Modifier.TextTransform TextTransform.Italic ] ]
+                [ str "Italic" ] ]
+          li [ ]
+            [ Text.span [ Modifiers [ Modifier.TextTransform TextTransform.Capitalized ] ]
+                [ str "capitalized" ] ]
+          li [ ]
+            [ Text.span [ Modifiers [ Modifier.TextTransform TextTransform.UpperCase ] ]
+                [ str "UpperCase" ] ]
+          li [ ]
+            [ Text.span [ Modifiers [ Modifier.TextTransform TextTransform.LowerCase ] ]
+                [ str "LowerCase" ] ]
+          li [ ]
+            [ Text.span [ Modifiers [ Modifier.TextTransform TextTransform.UpperCase
+                                      Modifier.TextTransform TextTransform.Italic ] ]
+                [ str "Italic & UpperCase" ] ] ]
+
+let weight () =
+    Content.content [ ]
+        [ li [ ]
+            [ Text.span [ Modifiers [ Modifier.TextWeight TextWeight.Light ] ]
+                [ str "Light" ] ]
+          li [ ]
+            [ Text.span [ Modifiers [ Modifier.TextWeight TextWeight.Normal ] ]
+                [ str "Normal" ] ]
+          li [ ]
+            [ Text.span [ Modifiers [ Modifier.TextWeight TextWeight.SemiBold ] ]
+                [ str "Semi bold" ] ]
+          li [ ]
+            [ Text.span [ Modifiers [ Modifier.TextWeight TextWeight.Bold ] ]
+                [ str "Bold" ] ] ]
+
+let view =
+    Render.docPage [
+        Render.contentFromMarkdown
+            """
+# Modifiers - Typography
+
+Change the **size** and **color** of the text for one or multiple viewport width.
```

Comment:
maybe add an example of changing the size on multiple viewports too

Reply:
Can't you let me be lazy ? 😠 

:trollface: :trollface: :trollface: 

Ok ok, I will add the missing section ^^

Reply:
Ok it's possible that thanks to the sample we avoid a bug in the next release :D 🤷‍♂️

Reply:
you can thank me appropriately later :-p